CNAME

```diff
@@ -1 +1 @@
-opensource.forter.com
\ No newline at end of file
+opensource.forter.dev
```

Comment:
```suggestion
forter.dev
```

Reply:
why not this?

Reply:
then `blog.forter.dev` later

Reply:
Github does not allow you to use a root DNS entry

Reply:
https://help.github.com/en/articles/setting-up-an-apex-domain ??